### PR TITLE
added custom class

### PR DESCRIPTION
**Pull request by orrybaram**
T: patch
R: @kesla @ellell @iefserge 

Adds the ability to add a custom class to the root div so we can style it differently.

### CODE REVIEW

**Comment by kesla** [comment]
@orrybaram hey tests.

**Comment by kesla** [comment]
CI is failing and we need tests for this change.

File: index.js [hunk 1]
```diff
@@ -10,7 +10,7 @@ export function initialState (props) {
 }
 
 export function render ({ props, state }, setState) {
-  const { items, onChange, removeOnHide } = props;
+  const { items, onChange, removeOnHide, customClass } = props;
```
Comment (iefserge) [inline index.js]: call it just `class` ? that way it'll be consistent with builtin elements

File: index.js [hunk 2]
```diff
@@ -38,7 +38,7 @@ export function render ({ props, state }, setState) {
   ));
 
   return (
-    <div class='tab-control'>
+    <div class='tab-control ${customClass}'>
```
Comment (iefserge) [inline index.js]: that should work too `class={['tab-control', customClass]}`

File: index.js (hunk 1 above)
Comment (kesla) [inline index.js]: +1 - it's also what we're using elsewhere

Comment (orrybaram) [inline index.js]: @kesla @iefserge im running into validation issues with semistandard when calling the prop `class` calling it anything else works fine.

**Comment by ellell** [comment]
Nice!

**Pull request merged**